ReactWindows/ReactNative/UIManager/BaseViewManager.cs

```diff
@@ -292,7 +293,7 @@ private DimensionBoundProperties GetOrCreateDimensionBoundProperties(TFrameworkE
             if (!_dimensionBoundProperties.TryGetValue(view, out properties))
             {
                 properties = new DimensionBoundProperties();
-                _dimensionBoundProperties.Add(view, properties);
+                _dimensionBoundProperties.AddOrUpdate(view, properties, (k, v) => v);
```

Comment:
> (k, v) => v [](start = 71, length = 12)

If the key already exists, it looks like it'll keep the old value rather than overwriting it. Is that what we want?

Reply:
Yeah, an interesting point. All the usages are of "Add" type, most of them embedded in an "if !present" block or guaranteed to be called once per view.
On the other hand the name of the method may suggest that using "latest" value is more correct, at least from a style p.o.v.

I'll update all occurrences.


---
In reply to: [170749392](https://github.com/Microsoft/react-native-windows/pull/1672#discussion_r170749392) [](ancestors = 170749392)

Reply:
maybe we should have an extension method that always overwrites. For example:

_dimensionBoundProperties.AddOrUpdate(view, properties);

---
In reply to: [170764226](https://github.com/Microsoft/react-native-windows/pull/1672#discussion_r170764226) [](ancestors = 170764226,170749392)